### PR TITLE
add(tools): Tailwind CSS Gradient Generator

### PR DESCRIPTION
**Pull request by khatabwedaa**
| Name                 | Link                 |
| -------------------- | -------------------- |
| _Tailwind CSS Gradient Generator_ | _https://tailwindcomponents.com/gradient-generator_ |

_Create perfect gradients for your Tailwind CSS sites easily with zero lines of code._

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,6 +219,7 @@
 - 🌍 [Tailwind Cheat Sheet](https://github.com/LeCoupa/awesome-cheatsheets/blob/master/frontend/tailwind.css) - Tailwind CSS class names in one single file.
 - 🌍 [Tailwind Cheat Sheet](https://umeshmk.github.io/Tailwindcss-cheatsheet) - Tailwind CSS class names, variants and directives cheat sheet.
 - 🌍 [Tailwind Cheat Sheet](https://tailwindcomponents.com/cheatsheet) - Tailwind CSS class names in a searchable page.
+- 🌍 [Tailwind Gradient Generator](https://tailwindcomponents.com/gradient-generator) - Create perfect gradients for your Tailwind CSS sites easily with zero lines of code.
```
Comment (innocenzi) [inline README.md:222]: It should have these emojis: 🎨🌍🔧
You can group it under `Palettolithic`

Can you rephrase the description so it's shorter and doesn't feel like a slogan?

https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md#formats-naming-conventions-and-descriptions

**Review by innocenzi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (khatabwedaa) [reply README.md]: Updated :+1:

**Review by innocenzi — APPROVED** [review]

**Comment by innocenzi** [comment]
Thank you!
